### DISCUSSION TITLE
How to integrate amp pages in nuxt 3.8

### DISCUSSION THREAD

**Discussion by manasigajbare88** [nuxt/nuxt#27046] [Questions]
hey team, we have recently migrated the project from nuxt 2 to nuxt 3.8 , as nuxtjs/amp is not compatible with nuxt 3. can anyone suggest any other way to integrate amp pages into nuxt 3.8

thank you.

**Comment by xjccc** [comment]
Have you been paying attention to this?

[nuxtjs/amp](https://github.com/nuxt-community/amp-module?tab=readme-ov-file)

![image](https://github.com/nuxt/nuxt/assets/16478776/0819726f-8f2d-4929-a9de-b7b2f6a074c4)

**Reply by manasigajbare88** [reply]
hey @xjccc thanks for reply will check this out
